src/components/button/button.twig

```diff
@@ -1,19 +1,36 @@
-{% set tagName = tagName | default(href ? 'a' : 'button') %}
+{% set tag_name = tag_name|default(href ? 'a' : 'button') %}
 
-<{{tagName}}
-  class="c-button{% if class %} {{class}}{% endif %}"
-  {% if ariaExpanded %}
-    aria-expanded="{{ariaExpanded}}"
+<{{ tag_name }}
+  class="c-button{% if class %} {{ class }}{% endif %}"
+  {% if aria_expanded %}
+    aria-expanded="{{ aria_expanded }}"
   {% endif %}
-  {% if tagName == 'a' %}
-    href="{{href|default('#')}}"
-  {% elseif tagName == 'button' %}
-    type="{{type|default('button')}}"
+  {% if tag_name == 'a' %}
+    href="{{ href|default('#') }}"
+  {% elseif tag_name == 'button' %}
+    type="{{ type|default('button') }}"
     {% if disabled %}
       disabled
     {% endif %}
-  {% endif %}>
+  {% endif %}
+>
+  {% block content_start %}
+    {% if content_start_icon %}
+      {% include '@cloudfour/components/icon/icon.twig' with {
+        name: content_start_icon,
+        inline: true
+      } %}
+    {% endif %}
+  {% endblock %}
   {% block content %}
-    {{label | default('Hello world')}}
+    {{ label|default('Hello world') }}
+  {% endblock %}
+  {% block content_end %}
+    {% if content_end_icon %}
+      {% include '@cloudfour/components/icon/icon.twig' with {
+        name: content_end_icon,
+        inline: true
+      } %}
+    {% endif %}
   {% endblock %}
-</{{tagName}}>
+</{{tag_name}}>
```

Comment:
This is technically a breaking change, and a quick search of the this and the theme repo suggests the property is in use. Do you think we could save changes to the case of existing property names for a PR that handles all of those breakages in one swoop instead of bumping this from a feature to a breaking release?

I think it's a good idea for your new properties to use snake case to begin with.

Reply:
Ah, good call. Thank you for catching this! 🔍 👀

Reply:
Done! :https://github.com/cloudfour/cloudfour.com-patterns/pull/1264/commits/030174785617be60b3fc2fb77f82c96dbf6ec5a8